### DISCUSSION TITLE
How to pass spaces in RESTIC_BACKUP_SOURCES?

### DISCUSSION THREAD

**Discussion by nonsavant** [djmaze/resticker#114] [Q&A] [answered]
Tried every possible escaping mechanism that I could think of and I cannot get RESTIC_BACKUP_SOURCES to accept an argument including a space.

`RESTIC_BACKUP_SOURCES: data/my folder`
`RESTIC_BACKUP_SOURCES: "data/my folder"`
`RESTIC_BACKUP_SOURCES: "'data/my folder'"`
`RESTIC_BACKUP_SOURCES: data/my\ folder`
`RESTIC_BACKUP_SOURCES: "data/my\ folder"`

No matter what I try, restic treats it as two separate backup sources, always delimiting on the space. E.g.

```
/data/my does not exist, skipping
folder does not exist, skipping
```

**Comment by djmaze** [comment, marked as answer]
Was fixed in #120 / v1.6.0
